### ISSUE TITLE
case insensitive sorting

### ISSUE THREAD

**Issue by pordonez** [rubanraj54/vue-bootstrap4-table#24]
Hi,

It appears that sorting is case sensitive so, for example, items starting with "Z" appear before those starting with "a" after sorting.  Can you add support for case insensitive sorting and make it configurable?

Thank you, Peter

**Comment by rubanraj54** [comment]
Hi @pordonez ,

Your idea really makes sense.

I'll include this feature in my next release..

Cheers,
Ruby.

**Comment by rubanraj54** [comment]
Hey @pordonez 

I've implemented this feature in my current branch. So you can use it from next release.

Cheers,
Ruby.

**Comment by rubanraj54** [comment]
Hey @pordonez ,

This feature is available from version 1.1.2. This is not yet documented in Readme.

Usage:

```
columns: [
    ...
    {
        label: "First Name",
        name: "first_name",
        sort: true,
        initial_sort: true, // "false" by default
        sortCaseSensitive: false, // "true" by default
    }
    ...
]
```

Cheers,
Ruby.
